cmake/partition_manager.cmake

```diff
@@ -177,7 +177,28 @@ add_region(
   DEVICE NRF_FLASH_DRV_NAME
   )
 
-if (CONFIG_PM_EXTERNAL_FLASH)
+dt_chosen(ext_flash_dev PROPERTY nordic,pm-ext-flash)
+if (DEFINED ext_flash_dev)
+  dt_prop(dev_name PATH ${ext_flash_dev} PROPERTY label)
+  dt_prop(num_bits PATH ${ext_flash_dev} PROPERTY size)
+  math(EXPR num_bytes "${num_bits} / 8")
+
+  add_region(
+    NAME external_flash
+    SIZE ${num_bytes}
+    BASE ${CONFIG_PM_EXTERNAL_FLASH_BASE}
```

Comment:
If user is already working in devicetree, couldn't the knowledge be present there ?

Reply:
No, this is old stuff that we might want to remove later. It has a bad name, and should be called "offset" or something as it can be used to hide the start of the external flash from the partition manager.

Reply:
So this information is not present in devicetree.